modules/api/current.js

```diff
@@ -90,6 +90,8 @@ module.exports = {
       attributes,
       rerun: this.isRerun,
       rerunOf: this.rerunOf,
+      ...(this.config.debug && { debug: this.config.debug }),
```

Comment:
The `debug` option is not a part of a launch request - it is a client option to see verbose debug logs, so please remove it from here and make sure that it is provided to the client class during creation.